### DISCUSSION TITLE
Crashing on EndeavourOS

### DISCUSSION THREAD

**Discussion by FlitPix** [LizardByte/community#34] [Sunshine] [answered]
Hello! I installed Sunshine from AUR and attempted to get it to work, and it does for a little while before eventually crashing with an error that I'm certainly not familiar with in any capacity: [log.txt](https://github.com/SunshineStream/Sunshine/files/8866352/log.txt)
This happens when using Sunshine with the Moonlight app on my Android phone.

The host running Sunshine is on a 1050 Ti and EndeavourOS. Any ideas? (As a heads up I am somewhat new to Linux as a whole so apologies if I am confused by some things. It's very likely I might have missed something in configuration!)

**Comment by KuleRucket** [comment, marked as answer]
It looks like you are losing the connection somehow so could be a network issue?

There is also a crash at the end of your log file after a few connections/disconnections but I think that this is a separate problem because I I also see this crash and my connection is otherwise stable. As a workaround for the crash, I auto restart the service using Restart=on-failure / RestartSec=5 in sunshine.service but that isn't going to solve your disconnection problem.

**Reply by FlitPix** [reply]
I'm intentionally disconnecting to test Sunshine, so that's not an issue; I'm just concerned about the crashes. If you're experiencing them too it's likely a bug?

**Reply by KuleRucket** [reply]
It's definitely a bug and should be reported, I just hadn't got around to collecting enough useful information about it so never raised it. If you can test it with verbose logging that could be useful.

**Reply by FlitPix** [reply]
I will get to this and create an issue, thank you for confirming! I will say that I have moved to Manjaro, though I'd imagine the crashes should still occur here as well.
